dev_guide/templates.adoc

```diff
@@ -50,20 +55,54 @@ set of labels to apply to every resource created by the template.
 
 <1> The name of the template
 <2> Optional description for the template
-<3> A list of parameters for the template
-<4> A list of labels to apply to resources
-<5> A list of resources to create
+<3> The icon that will be shown in the UI for this template
+<4> A list of tags that this template will have in the UI
+<5> A list of parameters for the template
+<6> A list of labels to apply to all resources
+<7> A list of resources to create
 
 ====
 
-== Creating a Configuration From a Template
-You can create a configuration from a template using the CLI or, if a template
+== Creating a list of resources from a Template
+You can create a list of resources from a template using the CLI or, if a template
 has been uploaded to your project or global template library, using the web
 console.
 
 Create a template JSON file, like the above example, then upload it with the
```

Comment:
we have have JSON or YAML, I'd suggest rewriting this:

Create a template file (JSON or YAML)...